test/wasi/README.md

```diff
@@ -0,0 +1,5 @@
+# WASI Tests
+
+Compile with clang and `wasm32-wasi` target. The clang version used must be
+built with wasi-sysroot. See `build.sh` in this directory for the exact
+commands used.
```

Comment:
Are these tests imported from somewhere, or are they freely written?

Reply:
they were imported from lucet, but have been somewhat rewritten and reformatted.